Header header
Header image_header
BoundingBox[] bounding_boxes
float64[] depth_array